src/components/statusBarRe.re

```diff
@@ -1,11 +1,5 @@
 external statusBar : ReasonReact.reactClass = "StatusBar" [@@bs.module "react-native"];
 
-/*barStyle::[ | `darkContent | `default | `lightContent]? =>
-  hidden::bool? =>
-  backgroundColor::string? =>
-  translucent::bool? =>
-  networkActivityIndicatorVisible::bool? =>
-  showHideTransition::[ | `fade | `none | `slide]*/
```

Comment:
Actually, shouldn't `make` contain a trailing `unit` arg? Because now the optionals cannot be erased.